taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestCancelledTaskRetrieval.java

```diff
@@ -131,7 +131,7 @@ public void deletion_of_camunda_process_instance_should_complete_taskana_task()
       Instant taskanaTaskCompletion = taskanaTasks.get(0).getCompleted();
       Instant taskanaTaskCreation = taskanaTasks.get(0).getCreated();
       TaskState taskanaTaskState = taskanaTasks.get(0).getState();
-      assertThat(TaskState.COMPLETED.equals(taskanaTaskState)).isTrue();
+      assertThat(TaskState.TERMINATED.equals(taskanaTaskState)).isTrue();
```

Comment:
Can't we use assertThat(taskanaTaskState).isEqualTo(TaskState.TERMINATED) ?

Reply:
Done